src/API/Endpoints/Reports/FormPerformance.php

```diff
@@ -10,39 +10,126 @@
 
 class FormPerformance extends Endpoint {
 
+	protected $payments;
+
 	public function __construct() {
 		$this->endpoint = 'form-performance';
 	}
 
-	public function get_report($request) {
+	public function get_report( $request ) {
+
+		// // Check if a cached version exists
+		// $cached_report = $this->get_cached_report( $request );
+		// if ( $cached_report !== null ) {
+		// Bail and return the cached version
+		// return new \WP_REST_Response(
+		// [
+		// 'data' => $cached_report,
+		// ]
+		// );
+		// }
```

Comment:
What's this? 😄

Reply:
Oops! Forgot to uncomment this block. Had commented out the caching while I was working to gurantee I was seeing fresh reports + the caching logic wasn't interfering. Fixed, ref: https://github.com/impress-org/givewp/commit/07d37d5956647343105270a04ee8ac94fe04e195